### PR TITLE
Introduce crds directory for compatibility with Helm v3

### PR DESCRIPTION
**Pull request by brunsgaard**
This commit introduce a crd directory for the helm chart which has all custom resource definitions. The files in the crd directory is plain YAML. The crds got the label `app.kubernetes.io/name: postgres-operator` and removes all the templating.

Helm v3 ignores the objects from the `templates` directory which have a `crd-install` hook. This commit addes templates/crds.yaml which generates YAMLs for CRDs. The hooks from these CRDs are detected by Helm v2 as well as v3. Helm v2 executes the hook and Helm v3 ignores the hook (YAML files are not applied)

The approach is inspired by the prometheus-operator chart
helm/charts@89b233eef6dbc1b6fac418bde3a5a6f4e14406d4

### CODE REVIEW

**Comment by brunsgaard** [comment]
This PR is an iteration on @rektide work in #735

**Comment by FxKu** [comment]
LGTM! Thanks @brunsgaard and @rektide for you efforts. I see, that quite some things changed with helm. Can I ask you to also update the [quickstart](https://github.com/zalando/postgres-operator/blob/master/docs/quickstart.md#helm-chart) for this PR, as some commands are outdated by now (`init`, `--name` etc)?

File: docs/quickstart.md [hunk 1]
```diff
@@ -73,22 +73,34 @@ manifest.
...
+LAST DEPLOYED: Wed Nov 27 13:14:37 2019
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+
+# Verify that the operator is running
+bash-3.2$ kubectl get pods -l "app.kubernetes.io/name=postgres-operator" --all-namespaces
+NAMESPACE   NAME                                READY   STATUS    RESTARTS   AGE
+default     postgres-operator-dbdd565d6-6l9hl   1/1     Running   0          104s
+```
```
Comment (FxKu) [inline docs/quickstart.md]: You can remove this paragraph. It's already covered later.

File: docs/quickstart.md [hunk 2]
```diff
@@ -73,22 +73,34 @@ manifest.
...
 
 ```bash
-# 1) initialize helm
-helm init
-# 2) install postgres-operator chart
-helm install --name zalando ./charts/postgres-operator
+helm install postgres-operator ./charts/postgres-operator --values ./charts/postgres-operator/values-crd.yaml
 ```
 
+The chart works with both helm2 and helm3. Documentation for installing
```
Comment (FxKu) [inline docs/quickstart.md]:
```suggestion
The chart works with both Helm 2 and Helm 3. Documentation for installing
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -73,22 +73,34 @@ manifest.
...
+```
+
+To use CRD-based configuration you need to specify the [values-crd yaml file](../charts/postgres-operator/values-crd.yaml).
 
 ```bash
-# 1) initialize helm
-helm init
-# 2) install postgres-operator chart
-helm install --name zalando ./charts/postgres-operator
+helm install postgres-operator ./charts/postgres-operator --values ./charts/postgres-operator/values-crd.yaml
```
Comment (FxKu) [inline docs/quickstart.md]: `--values` doesn't work in helm v3. But `-f` does (I think also in helm2).

Comment (brunsgaard) [reply docs/quickstart.md]: `--values` should still be fine https://github.com/helm/helm/blob/master/cmd/helm/install.go#L44
but I like `-f` better actually

**Comment by brunsgaard** [comment]
There you go.

**Comment by FxKu** [comment]
:+1:

**Comment by erthalion** [comment]
:+1:

**Pull request merged**